install/kubernetes/cilium/templates/spire/agent/daemonset.yaml

```diff
@@ -27,10 +27,14 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: {{ .Values.auth.mTLS.spire.install.agent.serviceAccount.name }}
-      {{- if gt (len .Values.auth.mTLS.spire.install.agent.initContainers) 0 }}
       initContainers:
-        {{- toYaml .Values.auth.mTLS.spire.install.agent.initContainers | nindent 8 }}
-      {{- end }}
+        - name: init
```

Comment:
so afaik changing `initContainers` is the thing that breaks users who don't follow proper upgrade methods, and instead just bump their image. I think the conclusion of talking about this recently was that if this needs to happen, it needs to be announced as loudly as possible.
cc @michi-covalent where do we announce this?

Reply:
This got added only in main not even in a snapshot release to this should not be an issue in the upgrade flow

Reply:
ah I didn't realise, fine by me then